src/vs/editor/contrib/unicodeHighlighter/unicodeHighlighter.ts

```diff
@@ -424,6 +516,78 @@ const DECORATION = ModelDecorationOptions.register({
 	}
 });
 
+export class DisableHighlightingOfAmbiguousCharactersAction extends EditorAction {
+	public static ID = 'editor.action.unicodeHighlight.disableHighlightingOfAmbiguousCharacters';
+	public readonly shortLabel = nls.localize('unicodeHighlight.disableHighlightingOfAmbiguousCharacters.shortLabel', '');
```

Comment:
It looks like `shortLabel` here is gettting removed by the tree shaking. The reason is that if you try to find all references or rename this property, you will notice that no references are found by TypeScript, or that the rename introduces a compilation error. 

The solution is to be more explicit (instead of relying 100% on duck-typing) and create a new interface `IDisableUnicodeHighlightAction` with the `shortLabel` property and then explicitly implement it.